pyarrow-integration-tests/src/pyarrow_integration_tests/__init__.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from pathlib import Path
+from typing import Optional
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pyarrow.parquet.encryption as pe
+from .kms import kms_client_factory
+
+
+def generate_test_files():
```

Comment:
I'm guessing it's just an oversight in PyArrow, but it looks like there's no way to configure PyArrow to use uniform encryption, you have to specify some column keys.

Reply:
Yeah, that sounds right. I remember hitting a similar error and getting a very unhelpful error message about it. In general, the pyarrow implementation feels fairly incomplete / janky to me.